### PR TITLE
feat(FEC-10804): add adapterData to getPlayback context request

### PR DESCRIPTION
**Pull request by Yuvalke**
### Description of the Changes

support adapterData for getPlaybackContext request

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: flow-typed/types/media-info.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ declare type OTTProviderMediaInfoObject = OVPProviderMediaInfoObject & {
   fileIds?: string,
   streamerType?: string,
   urlType?: string,
+  adapterData?: Array<Object>,
```
Comment (OrenMe) [inline flow-typed/types/media-info.js:15]:
```suggestion
  adapterData?: Array<{[key: string]: string}>,
```

File: flow-typed/types/playback-context.js [hunk 2]
```diff
@@ -4,5 +4,6 @@ declare type ProviderPlaybackContextOptions = {
   assetFileIds?: string,
   context?: string,
   streamerType?: string,
-  urlType?: string
+  urlType?: string,
+  adapterData?: Object
```
Comment (OrenMe) [inline flow-typed/types/playback-context.js:8]:
```suggestion
  adapterData?: Array<{[key: string]: string}>
```

**Review by OrenMe — CHANGES_REQUESTED** [review]
Also need to add to docs in kaltura player

File: flow-typed/types/media-info.js (hunk 1 above)
Comment (OrenMe) [reply flow-typed/types/media-info.js:15]: as this type repeats itself than better define a type and use in both places

Comment (Yuvalke) [reply flow-typed/types/media-info.js]: I'm not sure it's typed as value, objectType.

File: flow-typed/types/playback-context.js (hunk 2 above)
Comment (Yuvalke) [reply flow-typed/types/playback-context.js:8]: It's an object of an object,s not an array as mentioned on docs.

Comment (OrenMe) [reply flow-typed/types/playback-context.js:8]: see the api docs - unfourtunatly the jira tickets has invalid sample code

Comment (OrenMe) [reply flow-typed/types/playback-context.js:8]: or maybe not, LMC

Comment (OrenMe) [reply flow-typed/types/playback-context.js:8]:
```suggestion
  adapterData?: {[key: string]: {value: string, objectType?: string}}
```

Comment (OrenMe) [reply flow-typed/types/playback-context.js:8]: you are correct, the oroginal docs are invalid apperantly and ticket has correct form :-)

Comment (Yuvalke) [reply flow-typed/types/playback-context.js:8]: I'm not sure it'll be typed as you wrote, I don't think they have a strict structure

Comment (OrenMe) [reply flow-typed/types/playback-context.js:8]: it's an API - it has, or at least should have, a strict structure

**Review by OrenMe — APPROVED** [review]
@Yuvalke please update the kaltura-player loadMedia docs for OTT.

**Pull request merged**